li-utils/src/main/java/com/linkedin/common/urn/ModelUrn.java

```diff
@@ -0,0 +1,50 @@
+package com.linkedin.common.urn;
+
+import java.net.URISyntaxException;
+
+import com.linkedin.common.FabricType;
+
+import static com.linkedin.common.urn.UrnUtils.toFabricType;
+
+
+public final class ModelUrn extends Urn {
+
+  public static final String ENTITY_TYPE = "model";
+
+  private static final String CONTENT_FORMAT = "(%s,%s,%s)";
+
+  private final DataPlatformUrn platformEntity;
+
+  private final String modelNameEntity;
+
+  private final FabricType originEntity;
```

Comment:
what if the same model is trained or generated in multiple fabrics?

Reply:
In that case we would have a model per fabricType very similar to Datasets per fabric.